projects/demo/src/app/demo/pages/chip-page/chip-page.component.html

```diff
@@ -0,0 +1,60 @@
+<pa-demo-page>
+    <pa-demo-title>Chip</pa-demo-title>
+    <pa-demo-description>Small and compact chip used to display information, make selections, filter content or input visual content.
+    </pa-demo-description>
+
+    <pa-demo-examples>
+        <pa-chip noCloseButton>Not closable chip</pa-chip>
+        <pa-chip disabled>Disabled chip</pa-chip>
+        <ng-container *ngFor="let chip of chips">
+            <pa-chip *ngIf="chip.contact" [contact]="chip.contact" (closed)="closeChip(chip.name)">{{chip.name}}</pa-chip>
```

Comment:
this component should be agnostic, so the input should not be `contact` as you can pass any kind of text and avatar to the chip, doesn't have to be a user at all.

Reply:
so according to my readings about chips... this looks really standard, that's why I made this call.
would you call it nameAndImage ?